pkgs/leak_tracker/lib/src/shared/_util.dart

```diff
@@ -32,6 +32,13 @@ T cast<T>(value) {
   );
 }
 
+extension IterableExtensions<T> on Iterable<T> {
+  T? get onlyOrNull {
```

Comment:
This could use a doc. The name might need some work as well. It isn't obvious at first glance what this tries to accomplish.

Reply:
Added doc. The name is consistent with other names on iterable:

<img width="152" alt="Screenshot 2023-08-14 at 12 22 59 PM" src="https://github.com/dart-lang/leak_tracker/assets/12115586/fa465c14-c726-4414-9517-4a862855f83f">